packages/tools/devtools/devtools-core/src/data-visualization/DataEditing.ts

```diff
@@ -31,7 +32,7 @@ export interface Edit {
 	/**
 	 * Data contains the new data that will be edited into the DDS
 	 */
-	data: Serializable<unknown>;
+	data: Serializable<unknown> | Primitive;
```

Comment:
I believe this can probably be 
```suggestion
	data: Serializable<unknown> | null | undefined;
```

You'll have to disable the linter rule on this line, but that's fine. We need to support customer data that is `null`.